### ISSUE TITLE
Version 2.2.2 can not be build with "old" libexif versions

### ISSUE THREAD

**Issue by renchap** [tonytonyjan/exif#28]
The new tags added in https://github.com/tonytonyjan/exif/commit/26f9bac108584b2874b8e70edc80a7291f61d5b8 require a `libexif` 0.6.23 or newer.
This version is quite recent and not included in, for example, Debian and Ubuntu:

```
275.9 exif_entry_to_ivar.c:462:8: error: ‘EXIF_TAG_IMAGE_DEPTH’ undeclared (first use
275.9 in this function); did you mean ‘EXIF_TAG_IMAGE_LENGTH’?
275.9   462 |   case EXIF_TAG_IMAGE_DEPTH:
275.9       |        ^~~~~~~~~~~~~~~~~~~~
275.9       |        EXIF_TAG_IMAGE_LENGTH
```

This gem should check for either the `libexif` version, or (probably better) for the existence of the symbol.

**Comment by renchap** [comment]
Fixed in #29

**Comment by tonytonyjan** [comment]
FYI, v2.2.3 released 🎉
